utils/mock-data-loader/gef/constants/facility-provided-details.ts

```diff
@@ -0,0 +1,9 @@
+export const FACILITY_PROVIDED_DETAILS = {
```

Comment:
Any new constant or helpers should be under `/libs/common`, I believe some of them already exists there.

Reply:
These are not new constants - they are just converted from `javascript` to `typescript`. Many of the constants in mock data loader are redifined here so that they can be used in e2e tests (anything used by e2e tests must be a javascript file as our cypress setup is not configured to run ts files)

Reply:
`e2e-tests` have their own constants defined in JS, I can see these constants are used in `mock-data-loader` below unless I am missing something.

Reply:
`FACILITY_TYPE` constant commonised in #3359 - please review

Reply:
Thanks Josh -- And thanks for splitting this into a seperate PR unprompted, it made reviewing it a breeze!